doc/fuzzing.md

```diff
@@ -153,13 +153,17 @@ You may also need to take care of giving the correct path for `clang` and
 `clang++`, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
 `clang` does not come first in your path.
 
-Full configuration step that was tested on macOS with `brew` installed `llvm`:
+Currently the default linker on macOS `ld` does not work with recent versions of `llvm` so you will need to install llvm's linker `lld` and use it with the CMAKE_EXE_LINKER_FLAG `-fuse-ld=lld`.
```

Comment:
Rather than adding text that repeats the code below, we could just say something like: "Using lld is required to due to issues with Apples `ld` and LLVM".